Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:alpine AS build
+RUN apk add --no-cache git make
+WORKDIR /build
+COPY . .
+ENV CGO_ENABLED=0
+RUN make build
+
+FROM alpine:latest
```

Comment:
Can this just be a `scratch` container since it only runs the `yamlfmt` binary?

Reply:
Unfortunately, that is not possible due to the requirements of some ci-cd systems.
Gitlab CI for example needs to have at least a minimal shell environment inside of the image. See https://docs.gitlab.com/ee/ci/docker/using_docker_images.html#image-requirements for further details.

Reply:
Interesting, didn't know that. Thanks!